applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/response_data.h

```diff
@@ -0,0 +1,175 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Martin Fusseder, https://github.com/MFusseder
+//
+
+#if !defined(KRATOS_RESPONSE_DATA_H_INCLUDED )
+#define  KRATOS_RESPONSE_DATA_H_INCLUDED
+
+
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+
+
+namespace Kratos
+{
+    enum class TracedStressType
+    {
+        FX,
+        FY,
+        FZ,
+        MX,
+        MY,
+        MZ,
+        FXX,
+        FXY,
+        FXZ,
+        FYX,
+        FYY,
+        FYZ,
+        FZX,
+        FZY,
+        FZZ,
+        MXX,
+        MXY,
+        MXZ,
+        MYX,
+        MYY,
+        MYZ,
+        MZX,
+        MZY,
+        MZZ,
+        StressTypeNotAvailable
+    };
+
+    enum class StressTreatment
+    {
+        mean,
```

Comment:
Please be consistent with the upper or lowercase

Reply:
Thanks for the hint